sass/core.scss

```diff
@@ -2932,6 +2934,78 @@
   }
 }
 
+.yotpo-gallery {
+  &.yotpo {
+    .yotpo-modal {
+      opacity: 0;
+      &.yotpo-modal-active {
+        opacity: 1;
+        .yotpo-icon-left-arrow-thin, .yotpo-icon-right-arrow-thin {
+          cursor: pointer;
+          color: white;
+          font-size: 40px;
+          padding: 100px 0;
+          position: relative;
+          top: -100px;
+        }
+        .yotpo-icon-left-arrow-thin {
+          float: left;
+        }
+        .yotpo-icon-right-arrow-thin {
+          float: right;
+        }
+        .yotpo-icon.yotpo-icon-cross {
+          color: white;
+          z-index: 1000;
+        }
+        .gallery-arrows {
+          position: absolute;
+          top: 50%;
+          width: 100%;
+          padding: 0 20px;
+          height: 0;
+          z-index: 10001;
```

Comment:
doesn't take arrow height into consideration - margin-top: -20px (half the arrow height)